extensions/wikia/SpecialVideos/SpecialVideosHelper.class.php

```diff
@@ -130,7 +130,7 @@ protected function getTotalVideos( $videoParams ) {
 		wfProfileIn( __METHOD__ );
 
 		$mediaService = new MediaQueryService();
-		if ( $videoParams['sort'] == 'premium' ) {
+		if ( !empty( $videoParams['sort'] ) && $videoParams['sort'] == 'premium' ) {
 			$totalVideos = $mediaService->getTotalPremiumVideos();
 		} else if ( !empty( $videoParams['category'] ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "category" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)